x-pack/test/apm_api_integration/basic/tests/transaction_groups/error_rate.ts

```diff
@@ -46,24 +47,30 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           errorRateResponse = response.body;
         });
 
-        it('has the correct start date', async () => {
-          expect(first(errorRateResponse.erroneousTransactionsRate)?.x).to.be(1598439600000);
+        it('has the correct start date', () => {
+          expectSnapshot(first(errorRateResponse.erroneousTransactionsRate)?.x).toMatchInline(
+            `1598439600000`
+          );
         });
```

Comment:
Would it make sense to convert the timestamp to a human readable date?

```suggestion
        it('has the correct start date', () => {
          expectSnapshot(new Date(first(errorRateResponse.erroneousTransactionsRate)?.x)).toMatchInline(
            `Wed Aug 26 2020 13:00:00 GMT+0200 (Central European Summer Time)`
          );
        });
```

Reply:
great idea 👍🏻